packages/webkit2gtk_4_1.rb

```diff
@@ -162,7 +166,7 @@ def self.build
           -DUSER_AGENT_BRANDING='Chromebrew'"
     end
     @counter = 1
-    @counter_max = 5
+    @counter_max = 20
     loop do
       break if Kernel.system "#{CREW_NINJA} -C builddir -j #{CREW_NPROC}"
 
```

Comment:
Change `"DESTDIR=/usr/local/tmp/crew/dest ...` to `"DESTDIR=#{CREW_DEST_DIR} ...`?

Reply:
done